src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -637,8 +637,9 @@ private void InitializePackagesFolderInfo(string packagesFolderPath, bool exclud
             var tasks = new List<Task<IEnumerable<NuGetProjectAction>>>(maxTasks);
             var nugetActions = new List<NuGetProjectAction>();
 
-            var buildIntegratedProjects = nuGetProjects.OfType<BuildIntegratedNuGetProject>().ToList();
-            var nonBuildIntegratedProjects = nuGetProjects.Except(buildIntegratedProjects).ToList();
+            var nuGetProjectList = nuGetProjects.ToList();
```

Comment:
You have this issue because you rebased your branch but not the target branch.

Reply:
Thanks! Rebased target feature branch. Refreshed the base on this page.